ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/FixtureExtensions.kt

```diff
@@ -32,31 +30,6 @@ fun ContainerFixture.fillSingleTextField(text: String) = step("Fill single text
     Thread.sleep(1000)
```

Comment:
Not all text fields will require this, I would move it into file browser fixture

Reply:
This isn't specifically needed for the file browser, but I'll see if I can remove it